client/components/web-preview/style.scss

```diff
@@ -34,6 +34,7 @@
 		right: 0;
 		bottom: 0;
 	display: block;
+	background: fade-out( $gray-light, .2 );
```

Comment:
I would keep the `fade-out` here, as it uses the `$gray-light` variable where rgba values would need to be copied as numbers and thus can easily come out of sync.